owncloudApp/src/main/java/com/owncloud/android/presentation/ui/sharing/fragments/ShareFileFragment.kt

```diff
@@ -101,7 +107,7 @@ class ShareFileFragment : Fragment(), ShareUserListAdapter.ShareUserAdapterListe
     /**
      * List of public links bound to the file
      */
-    private var publicLinks: ArrayList<OCShare>? = null
+    private var publicLinks: ArrayList<OCShareEntity>? = null
```

Comment:
as I said before, thinking in ArrayList instead of List is not best, use interfaces instead